### DISCUSSION TITLE
Streamdeck internal websocket server in OBS Studio

### DISCUSSION THREAD

**Discussion by DhSufi** [elgatosf/elgato#150] [Help]
Hello, I noticed Streamdeck runs its own websocket server in OBS Studio: 
```
<StreamDeck> Plugin version 5.5.3.1
<StreamDeck> [Server] Listening on '127.0.0.1:28186'.
```

I was wondering if it is possible to connect to that websocket from a Stream Deck plugin and send/receive mesasges to OBS Studio.
Currently I am following the SDK [documentation](https://docs.elgato.com/streamdeck/sdk/references/websocket/plugin) to learn how to develop streamdecks plugins. I managed to use several fucntions (setImage, setTitle, etc) but I am not skilled enough to figure it out. I know it is advanced techniques but I would appreciate any example or directions where to look for information on the subject.

Thanks for your time.

**Comment by tiptronic** [comment]
OBS has a built-in websocket server you can connect to.
Best is to check at their forums.
